Makefile

```diff
@@ -64,6 +64,14 @@ deploy:
 	cd config/manager && kustomize edit set image controller=${IMG}
 	kustomize build config/default | kubectl apply -f -
 
+protoc-docker:
+	docker build - -t protoc < hack/grpc-protoc.Dockerfile
+
+protobuf: protoc-docker
+	docker run -v `pwd`:/eco -w /eco protoc:latest -I=api/proxy --go_out=plugins=grpc:api/proxy api/proxy/proxy.proto
+
+verify-protobuf:
+	./hack/verify.sh make -s protobuf
```

Comment:
I tried this Make target on my Fedora laptop and got this error:

```
Step 9/10 : COPY --from=go-plugin-builder /go/bin/protoc-gen-go /usr/local/bin/
 ---> 5e5be82d33b3
Step 10/10 : ENTRYPOINT ["/usr/local/bin/protoc"]
 ---> Running in cf63fc7f8c01
Removing intermediate container cf63fc7f8c01
 ---> 6ceda6b59828
Successfully built 6ceda6b59828
Successfully tagged protoc:latest
Could not map to virtual file: api/proxy/proxy.proto: Permission denied
make[1]: *** [Makefile:71: protobuf] Error 1
make: *** [Makefile:74: verify-protobuf] Error 2

```

If it works in Circlt CI, I don't mind if we merge this and fix this in a follow up branch.

Reply:
I think this was a problem with my Fedora 31 + docker configuration.

I installed podman instead and the script works.